### DISCUSSION TITLE
Planter spam

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#13557] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Despite the plant "nerfs" a while back it's still cheap and easy and there's no restriction to planters so people tend to spam a ton of them in the engine room 

Might be worth taking a look at plants again, specifically:
- No restrictions physical or otherwise on spamming planters
- Plants never actually dying from fire (at all) or water (takes 15 minutes of being submerged)

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/b568a91f-1f52-426e-8e76-701924a14e4c)

Could for example remove oxygen from a room *and* require oxygen (although balancing this is almost impossible due to how oxygen generator distribution works)

### Version

v1.2.8.0 (Winter Update hotfix 2)

**Comment by NotWendy** [comment]
I suggest increasing the growth rate of plants by a lot (not fruit production rate, but how long it takes for a plant to mature), so you can replant them easier, but making them more vulnerable to environmental hazards and limiting the amount of planters you can have on one submarine to 2.

Fully grown plants take up a ton of space and since balance-wise, there's no reason not to have as many as possible, submarines gradually turn into forests.

**Comment by bookslayer10** [comment]
I think they should just be more vulnerable to water damage and fire. I already don't even bother with them due to risk of losing the investment. It's fine if people spam a bunch of them, they already have upkeep costs from watering, just make it harder to keep them all alive through floods and the like.

**Comment by BlackMoons** [comment]
Should just restrict planters so they must be at least 2m apart from each other.
